scripts/ts-utils/setup_enclave.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+# Also need to set the port
+
+cd ../scripts/ts-utils/
+yarn install
+yarn tsc
+yarn node setup-enclave.js  $ENCLAVE_ACCOUNT $NEW_MRENCLAVE $SCHEDULE_UPDATE_BLOCK
```

Comment:
`node setup-enclave.js` => `npx ts-node setup-enclave.ts` It should work if you modify the code like this.

Reply:
Done ✅

Reply:
Thanks! 3